test/python/WMCore_t/Services_t/pycurlFileUpload_t.py

```diff
@@ -44,7 +44,7 @@ def testFileUpload(self):
         self.assertEqual(open(uploadedFilename).read(), open(fileName).read())
         # delete the uploaded file
         os.remove(uploadedFilename)
-        self.assertTrue('Success' in res)
+        self.assertTrue(b'Success' in res)
```

Comment:
This means that there is a change in the interface. `self.requestHandler.uploadFile` returns a list of bytes string in both py2 and py3 and not a list of "native" string (bytes in py2, uniode in py3)

Reply:
Nothing to discuss really, just something that we should be aware of and maybe report somewhere. in the function docstring? in the PR description?

Reply:
Yes, it's probably a good idea to mention it in the method docstring. Can you please do so?

Reply:
thanks! added to the docstring!